Tabloid/client/src/modules/TagManagment.js

```diff
@@ -3,15 +3,33 @@ const baseUrl = 'api/Tag';
 
 
 
-export const addTag = (tag) => {
-  return fetch(baseUrl, {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
-    },
-    body: JSON.stringify(tag)
-  });
-};
+// export const addTag = (tag) => {
+//   return fetch(baseUrl, {
+//     method: "POST",
+//     headers: {
+//       "Content-Type": "application/json",
+//       Authorization: `Bearer ${token}`
+//     },
+//     body: JSON.stringify(tag),
+//   });
+// };
```

Comment:
Do we need this commented code here still?

Reply:
@SchmidtFred  deleted the committed code